apps/web/src/App.tsx

```diff
@@ -39,6 +40,8 @@ const defaultQueryFn = async ({ queryKey }: { queryKey: string }) => {
   return response.data?.data;
 };
 
+console.log('Hi');
```

Comment:
Did the eslint did not worked for you with no-console checks? or did you miss that one? 🙃

Reply:
😅  You are very correct hhh, it's a warn level, should be throwing in CI, but it didn't. Good catch!

Reply:
@djabarovgeorge Fixed :)